src/address.ts

```diff
@@ -159,15 +159,32 @@ export class Address {
       '{{name.lastName}}{{address.citySuffix}}',
     ];
 
-    if (!format && this.faker.definitions.address.city_name) {
+    if (this.faker.definitions.address.city_name != null) {
       formats.push('{{address.cityName}}');
     }
 
-    if (typeof format !== 'number') {
-      format = this.faker.datatype.number(formats.length - 1);
-    }
+    const useFormat = (() => {
+      if (typeof format === 'number') {
+        deprecated({
+          deprecated: 'address.city(number)',
+          since: 'v6.2.0',
```

Comment:
We are already targeting v6.3.0 as next

```suggestion
          since: 'v6.3.0',
```

Is there also a test that spies for this warning?

Reply:
version update in 0747cb4aa2a2d0b6e49650019a018767ab8b29eb
tests in 43c3d31c5a924826fe05a4a05297d2fbf736eebc